homeassistant/components/climate/wink.py

```diff
@@ -33,6 +33,22 @@
 SPEED_MEDIUM = 'medium'
 SPEED_HIGH = 'high'
 
+HA_STATE_TO_WINK = {STATE_AUTO: 'auto', STATE_ECO: 'eco',
+                    STATE_FAN_ONLY: 'fan_only',
+                    STATE_HEAT: 'heat_only', STATE_COOL: 'cool_only',
+                    STATE_PERFORMANCE: 'performance',
+                    STATE_HIGH_DEMAND: 'high_demand',
+                    STATE_HEAT_PUMP: 'heat_pump',
+                    STATE_ELECTRIC: 'electric_only',
+                    STATE_GAS: 'gas', STATE_OFF: 'off'}
+WINK_STATE_TO_HA = {'auto': STATE_AUTO, 'eco': STATE_ECO,
```

Comment:
```python
WINK_STATE_TO_HA = {value: key for key, value in HA_STATE_TO_WINK.items()}
```

Reply:
😮 mind blown. Thanks. Didn't even think about that.